app/components/userOnboarding/index.tsx

```diff
@@ -0,0 +1,73 @@
+import { useState } from 'react';
+import UserCalendarDetails from './UserCalendarDetails';
+import UserDetails from './UserDetails';
+import { Button } from '../Button';
+
+const UserOnboarding = () => {
+  const [page, setPage] = useState<number>(0);
+  const FormTitlesAndSubtitles = [
+    {
+      title: 'Welcome to HapDay',
+      subtitle:
+        'Tell us a bit about yourself, we will need this to get your profile setup. You’ll be able to edit this later.',
+    },
+    {
+      title: 'Connect your calendar',
+      subtitle:
+        'Connect your calendar to automatically check for busy times and new events as they’re scheduled.',
+    },
+  ];
+
+  const PageDisplay = () => {
+    if (page === 0) {
+      return <UserDetails />;
+    }
+    if (page === 1) {
+      return <UserCalendarDetails />;
+    }
+    return '';
+  };
```

Comment:
Would it not be better to use two separate routes rather than rendering a component depending on the value in `useState`?

Reply:
In future we might use global state for this, zustand or redux, I have seen this pattern in other products as well for onboarding/large features with multiple screens